xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,415 @@
+    roles:-->
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      A virtual machine can run a full cluster stack. In this case, a cluster
+      node is not managed by the cluster.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The cluster can manage a virtual machine as a resource without being
+      aware of the services running inside the virtual machine. In this
+      scenario, the virtual machine is opaque to the cluster.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      A virtual machine can act as a cluster resource and run the
+      &pmremote; service daemon.
+      This allows the cluster to manage services running on the virtual
+      machine. In this situation, the virtual machine is transparent to
+      the cluster.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.install">
+  <title>Installation</title>
+  <para>Bascically, the installation consists of these steps:</para>
+  <orderedlist>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.phys.host"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.kvm.guest"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.integrate.guestsintocluster"/></para>
+   </listitem>
+  </orderedlist>
+
+  <sect2 xml:id="sec.ha.pmremote.config.phys.host">
+   <title>Configuring the Physical Host</title>
+   <para>
+    In this guide, the hostname <systemitem class="domainname">&wsI;</systemitem>
+    is used for the physical host. On <systemitem
+     class="domainname">&wsI;</systemitem> do the following:
+   </para>
+   <procedure>
+    <!--<title>Configuring the Physical Host</title>-->
+    <step>
+     <para>Install your cluster as described in the <citetitle>&instquick;</citetitle>.
+      <!--<xref linkend="art.ha.install.quick"/>-->
+     </para>
+     <remark>toms 2017-03-23: what about firewall settings? Should we open TCP
+     ports 2224, 3121, and 21064, and UDP port 5405?</remark>
+    </step>
+    <step>
+     <para>
+      Install the packages <package>acemaker-remote</package>
+      and <package>resource-agents</package>.</para>
+    </step>
+    <step>
+     <para>Configure the cluster nodes:</para>
+     <remark>toms 2017-03-23: Should we add all remote nodes into csync.cfg and /etc/hosts ?</remark>
+     <substeps>
+      <step>
+       <para>Allow cluster-related services through the local firewall on all
+        cluster nodes.</para>
+      </step>
+      <step>
+       <para>Create an authentication key to all cluster and remote
+        nodes:</para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0755 /etc/pacemaker
+&prompt.root;<command>dd</command> if=/dev/urandom of=/etc/pacemaker/authkey bs=4k count=1</screen>
+       <remark>toms 2017-03-03: is on SLEHAE the file /etc/corosync/authkey being used?</remark>
```

Comment:
There are two different authkeys.  Both are needed on corosync nodes, but only `/etc/pacemaker/authkey` is needed on remote nodes.